index.html

```diff
@@ -9,71 +9,209 @@
   </head>
 
   <body class="app-grid">
-      <h1 class="quiz-app-heading">Quiz-App</h1>
+    <h1
+      class="quiz-app-heading"
+      aria-describedby="Starting Page of the Quiz App showing first quiz-cards"
+    >
+      Quiz-App
+    </h1>
 
     <main class="app-grid__scrollarea">
+      <!--input component quiz-app-card-->
+      <!--first visible card-->
       <section class="quiz-app-card">
-        <button class="quiz-app-card__bookmark"></button>
+        <button
+          class="
+            quiz-app-card__bookmark
+            shadow
+            quiz-app-card__bookmark--activated
+          "
+        >
+          <img
+            src="../images/icons8-bookmark.svg"
+            alt="bookmark icon"
+            width="30"
+            height="30"
+          />
+        </button>
         <h2 class="quiz-app-card__heading">Question:</h2>
         <article class="quiz-app-card-question-container">
-          <details class="quiz-app-card-question">
-            <summary>show answer</summary class="quiz-app-card-question-button">
+          <p class="quiz-app-card-question">
```

Comment:
so far really nice ideas 👍  the naming 😹  here i would stay in class names like before...`quiz-app-card__question`

Reply:
and i like the idea to wrap the answer and show answer in an seperated article....you did because it´s changeable??? i would wrap the whole card around an article...but as i´ve said i like your idea also ;)